### ISSUE TITLE
Edit Game Details Window Problem with saving scores when saving by pressing enter

### ISSUE THREAD

**Issue by V0idhead** [JosefNemec/Playnite#1918]
**Describe the bug**
When a score field (User Score, Critic Score, Community Score) has the focus and the user presses enter to save the contents of that field are not saved.

**To Reproduce**
1. Open the Edit Game Details Screen for any game
2. With the mouse click into the User Score field and type anything
3. press Tab to go to the Critic Score field
4. Type anything
5. Press Enter to Save

Playnite saves the content of the User Score field, but not the Critic Score field.
All score fields seem to have this problem. If you use Tab to navigate into the field the contents are not saved unless you navigate out of first.

**Diagnostics ID**
40e0ad6f-1d2c-4c0c-8580-4af70afa9047

**Comment by darklinkpower** [comment]
Duplicate of https://github.com/JosefNemec/Playnite/issues/1878 (although your issue is more detailed)

**Comment by JosefNemec** [comment]
It's no the duplicate of #1878.

**Comment by darklinkpower** [comment]
You are right, sorry. I thought it created a dialogue too but I was wrong.

**Comment by chocolatechipcats** [comment]
I noticed a similar issue also happens in the Links section.

Steps to replicate:
1. Press F3 to open the Edit Game Details window.
2. Using keyboard navigation, go to the Links tab and to one of the title fields.
3. Delete "Store Page" and replace it with Steam/GOG.
4. Press enter to close the window.

It does save if I navigate to the next field before pressing enter.
